### PR TITLE
Feature/deeplinks

### PR DESCRIPTION
**Pull request by arne-z**

### CODE REVIEW

File: packages/app/__test__/index.test.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-import { render, screen } from '@testing-library/react';
-import Index from '../pages/index';
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import Index from "../pages/[network]/index";
 
-describe('Index page', () => {
-  it('should render elements properly', () => {
+describe("Index page", () => {
+  it("should render elements properly", () => {
```
Comment (RedVeil) [inline packages/app/__test__/index.test.tsx:6]: @arne-z is this supposed to be included here?

File: packages/app/hooks/staking/usePopLocker.ts [hunk 2]
```diff
@@ -18,6 +17,7 @@ export default function usePopLocker(address: string): SWRResponse<PopLockerMeta
       PopLocker__factory.connect(address, signerOrProvider),
     [chainId, address, signerOrProvider],
   );
+  console.log("popLocker", popLocker);
```
Comment (RedVeil) [inline packages/app/hooks/staking/usePopLocker.ts:20]:
```suggestion
```

File: packages/app/next.config.js [hunk 3]
```diff
@@ -3,17 +3,40 @@ const { join } = require("path");
...
   env: {
     RPC_URL: process.env.RPC_URL,
     CHAIN_ID: process.env.CHAIN_ID,
+    DEFAULT_CHAIN: process.env.DEFAULT_CHAIN,
     INFURA_PROJECT_ID: process.env.INFURA_PROJECT_ID,
+    CHAIN: process.env.DEFAULT_CHAIN,
     IS_DEV: process.env.IS_DEV,
   },
   poweredByHeader: false,
+  async rewrites() {
```
Comment (RedVeil) [inline packages/app/next.config.js:20]: Thats great! didnt know you could do this

**Comment by arne-z** [comment]
@RedVeil If you have any suggestions on copy, specifically when it comes to the Modal let me know.

File: packages/utils/src/getStakingPool.ts [hunk 4]
```diff
@@ -43,7 +43,7 @@ export async function getStakingPool(
 }
 
 export async function getPopLocker(key: string, popLocker: PopLocker, account?: Address): Promise<PopLockerMetadata> {
-  const tokenAddress = await popLocker.stakingToken();
+  const tokenAddress = await popLocker.callStatic.stakingToken();
```
Comment (RedVeil) [inline packages/utils/src/getStakingPool.ts:46]:
```suggestion
  const tokenAddress = await popLocker.stakingToken();
```

**Review by RedVeil — APPROVED** [review]

**Pull request merged**